src/Features/Core/Portable/ConvertAutoPropertyToFullProperty/AbstractConvertAutoPropertyToFullPropertyCodeRefactoringProvider.cs

```diff
@@ -96,8 +96,12 @@ internal bool IsValidAutoProperty(SyntaxNode property, IPropertySymbol propertyS
                 DeclarationModifiers.From(propertySymbol),
                 propertySymbol.Type, fieldName,
                 initializer: GetInitializerValue(property));
-            var containingType = GetTypeBlock(
-                propertySymbol.ContainingType.DeclaringSyntaxReferences.FirstOrDefault().GetSyntax(cancellationToken));
+
+            //var containingType = GetTypeBlock(
+            //    propertySymbol.ContainingType.DeclaringSyntaxReferences.FirstOrDefault().GetSyntax(cancellationToken));
+            var containingType = propertySymbol.ContainingType.DeclaringSyntaxReferences
```

Comment:
SuperNit (tm) containing type makes me think of SemanticModel.EnclosingNamedType, so maybe call this "typeDeclaration" or something?

Reply:
:+1: